### ISSUE TITLE
Kernel dies when trying to plot after fitting NN

### ISSUE THREAD

**Issue by thorrester** [keras-team/keras#13108]
Any time I train a NN (using tensorflow with keras) and attempt to plot (matplotlib) the loss history of the fit model, the kernel dies. I do not think it is code because running different code from different validated sources (links below) causes the same problem.

Also, it appears to be specific to tensorflow/keras and matplotlib. If I run an sklearn model and then plot it works fine.

example links:

https://github.com/chrisalbon/notes/blob/master/docs/deep_learning/keras/visualize_loss_history.ipynb

https://machinelearningmastery.com/tutorial-first-neural-network-python-keras/

Tried fixes: 
1. Restarting comp 
2. Removing keras and matplotlib 
3. Rolling back matplotlib to a previous version (3.02) 
4. Updating python 3.6 to 3.71 
5. Uninstalling python and anaconda from comp and re-installing 6. Running code in different browser (safari and chrome)

**Important**
I believe it has something to do with my installation. I sent the notebook and data to someone else with the same exact comp and setup and it worked fine. However, this isn't an isolated incident, as its been reported on stack before.

https://stackoverflow.com/questions/55833399/unable-to-plot-after-keras-fit-called

I've also tried running the py file through the command line to retrieve errors, but nothing happens (no error, no indication that the file is running). Other py files run fine though.

current versions

python - 3.71 
matplotlib - 3.0.3 
keras - 2.2.4 
tensorflow - 1.13.1

**Comment by gowtham-kp** [comment]
@thorrester I just ran the example and I am not getting any error. I suspect the problem is in installation . Can you please clean your environment and reinstall Keras and Tensorflow and see if it works. Thanks!

**Comment by thorrester** [comment]
Seemed to work! Thanks!
